tests/test_coronagraphy.py

```diff
@@ -175,3 +175,33 @@ def test_lyot_coronagraph():
 	# Checks performance of the coronagraph and if the focal length does not introduce artifacts
 	assert (wf_foc.power.max() / norm) < 5e-3
 	np.testing.assert_allclose(wf_foc.power, wf_foc2.power)
+
+
+def test_knife_edge_coronagraph():
+	grid = make_pupil_grid(64, 1.1)
+	aperture = circular_aperture(1)(grid)
+
+	focal_grid = make_focal_grid(q=5, num_airy=3)
+
+	prop = FraunhoferPropagator(grid, focal_grid)
+	lyot_stop = circular_aperture(0.95)(grid)
+
+	wf = Wavefront(aperture)
+	wf.total_power = 1.0
+	norm = prop(wf).power.max()
```

Comment:
Very minor: this should probably contain the Lyot stop as well. Totally unimportant for this test, but if you're making changes anyway, let's do this one as well.

Reply:
You want to normalize the PSF with respect to the Lyot Stop PSF?

Reply:
Of course, since that's what the planet looks like. And it's a coronagraphic image, not a PSF.

Reply:
Done